build.gradle.kts

```diff
@@ -131,6 +136,36 @@ tasks {
 		group = MAIN
 	}
 	
+	val buildInstaller by creating(com.install4j.gradle.Install4jTask::class) {
```

Comment:
Please add an import for the task

Reply:
4d289a10a7a5780eb7dd587f7de39b2761fe4be5